platform/podjs/bin/genPodjs.js

```diff
@@ -0,0 +1,45 @@
+const fs = require("fs");
+
+function executeReplacement(podJs, manifestJsonPath) {
+    try {
+        const manifestData = fs.readFileSync(manifestJsonPath, "utf8");
+        const podJsCode = fs.readFileSync(podJs, "utf8");
+
+        const pattern = new RegExp("window.manifestData", "g");
+        const replacedCode = podJsCode.replace(pattern, manifestData);
+
+        fs.writeFileSync(podJs, replacedCode);
+    } catch (err) {
+        console.error("Error while replacing code", err);
+    }
+}
+
+function copyPodJs(dest) {
+    if (!fs.existsSync(dest)) {
+        fs.mkdirSync(dest);
+
+        fs.copyFileSync(
+            "node_modules/@polypoly-eu/podjs/dist/pod.js",
```

Comment:
I think we may still be able to get the script's location from `argv`, and then find the `pod.js` file relative to that. That way we wouldn't have to hard code the specific location of the module in each feature, that's a bit error prone. But good enough for now I think.

Reply:
... and would thus eliminate the dependency.

Reply:
you mean the "node_modules/@polypoly-eu/podjs/dist/pod.js" path? And to pass this as an extra arg? Or the `${dest}/pod.js` one?
Cause from the current args we can only get where rollup lies... like: '.../polyPod/node_modules/.bin/rollup' 

not sure what u mean by dependency here @JJ

Reply:
If podjs is not added as a local dependency, that path will simply not exist locally.

Reply:
The podjs dep was already there, I think. So this is not about this PR, but a different story. Let's leave it this way. Thanks!